README.md

```diff
@@ -32,6 +32,7 @@ These functions act as stream processors, structuring Kinesis log records, which
 Parsers exist for the following services:
 - MySQL RDS slow query log
 - PostgreSQL RDS logs
+- SNS events
```

Comment:
Thought I'd add a bit to the README while I'm at it, but I'm not sure if this is a completely accurate description of what we added

Reply:
That seems fine, really :)